src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/FontCache/FontCacheUtil.cs

```diff
@@ -661,7 +661,11 @@ private static string NormalizeFontFamilyReference(string fontFamilyReference, i
             {
                 // No fragment separator. The entire string is a family name so convert to uppercase
                 // and add a fragment separator at the beginning.
+#if NET
                 return string.Concat("#", fontFamilyReference.AsSpan(startIndex, length)).ToUpperInvariant();
```

Comment:
Surprised System.Memory package didn't expose this?

Reply:
It's not about System.Memory exposing this I believe since the problem is a missing overload of string.Concat that takes spans and strings together or a missing type cast. Don't 100% remember which was the problem. May be solvable in a way that minimizes code impact though instead of just brute forcing it like I did.

Reply:
System.Memory doesn't have extension statics - but we're getting a bunch of new types of extensions in C# 14.  That could help reduce churn in places like this.